packages/origin-ui-core/src/components/admin/AdminUserForm.tsx

```diff
@@ -77,11 +77,9 @@ export function AdminUserForm(props: IProps) {
         dispatch(setLoading(true));
 
         try {
-            const formData: UserUpdateData = {
-                ...values
-            };
-
-            await adminClient.updateUser(values.id.toString(), formData);
+            await adminClient.updateUser(values.id, {
```

Comment:
Admin is now prohibited to change User data ?

Reply:
This is a good point. Somehow I missed this because UpdateUserDTO only contained the notifications and blockchain account, but we forwarded the whole user object and it accidentally worked, but it shouldn't have:
https://github.com/energywebfoundation/origin/pull/1934/files#diff-f45e6838c1a09e962293d393c9c0b7260b0bfdfbbbcd2600670479017c71c000R4

Will retest this and make sure it's working as expected.

Reply:
Done. Fixed it!